README.md

```diff
@@ -68,16 +68,16 @@ By default you will be building the latest unstable version of Julia. However, m
 
 Now run `make` to build the `julia` executable. To perform a parallel build, use `make -j N` and supply the maximum number of concurrent processes. (See [Platform Specific Build Notes](https://github.com/JuliaLang/julia#platform-specific-build-notes) for details.)
 When compiled the first time, it will automatically download and build its [external dependencies](#required-build-tools-and-external-libraries).
-This takes a while, but only has to be done once. If the defaults in the build do not work for you, and you need to set specific make parameters, you can save them in `Make.user`. The build will automatically check for the existence of `Make.user` and use it if it exists.
+This takes a while, but only has to be done once. If the defaults in the build do not work for you, and you need to set specific make parameters which can be saved in `Make.user`. The build will automatically check for the existence of `Make.user` and use it if it exists.
```

Comment:
The first part of this sentence is still in the active voice, so the second part should parallel that rather than switching to the passive voice. You could rewrite the whole sentence, but I think this is probably more easily readable in the active voice.